source/scss/_pf-component-card.scss

```diff
@@ -21,9 +28,76 @@
 
   .card-footer{
     background: $pf-card-footer-bg;
-    border-top-color: $pf-card-border-color
+    border-top-color: $pf-card-border-color;
   }
 }
+.card-title {
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+.pf-card__block,
+.pf-card__header  {
+  padding: $pf-spacer-lg;
+}
+.pf-card__footer  {
+  padding: $pf-spacer-sm $pf-spacer-lg;
+}
+.pf-card__footer p {
+  margin: $pf-spacer-xxxs 0;
+}
+.pf-card__header__title {
+  margin: 0;
+}
+.pf-card__header__dropdown {
+  margin: -$pf-spacer-xxs 0;
+}
+.pf-card__kebab {
+  margin: -$pf-spacer-xxs;
+}
+.pf-card__top-element {
+  margin-bottom: $pf-spacer-md;
+}
 .pf-card--accented {
   border-top: 2px solid $color-pf-blue-300;
 }
+.pf-card--select {
+  border: 2px solid transparent;
+  background-clip: content-box;
+  &:hover::after {
+    @include box-shadow($pf-box-shadow-md);
+  }
+}
+.pf-card--select--active {
```

Comment:
Bootstrap uses the class `.active` to define all its active state. I know @srambach hates this idea, but I'd like to keep consistency on that one. Can we change it?